### PR TITLE
DD4.16.3 release notes

### PR DESCRIPTION
**Pull request by aevesdocker**

### CODE REVIEW

File: desktop/release-notes.md [hunk 1]
```diff
@@ -24,13 +24,76 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+            <i class="fa fa-chevron-down"></i>
+          </a>
+        </h5>
+      </div>
+      <div id="collapseOne" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne">
+        <div class="panel-body">
+          <a class="btn btn-primary" href="https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-win-amd64" role="button">Download file</a> 
+        <br>
+        <br>
+        <b>Checksum:</b>
```
Comment (aevesdocker) [inline desktop/release-notes.md:48]: To do

File: desktop/release-notes.md [hunk 2]
```diff
@@ -24,13 +24,76 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+    </div>
+    </div>
+>
+> For the latest release of Docker Desktop for Mac and Linux, see [4.16.2](#4162)
+
+### Bug fixes and enhancements
+
+#### For Windows
+
+
```
Comment (aevesdocker) [inline desktop/release-notes.md:60]: For you @mat007 - what needs to be added here?

Comment (mat007) [inline desktop/release-notes.md:60]: Something like:
```suggestion
- Fixed Docker Desktop failing to start the WSL 2 backend when file system compression or encryption is enabled on `%LOCALAPPDATA%`. Fixes [docker/for-win#13184](https://github.com/docker/for-win/issues/13184).
- Fixed Docker Desktop failing to report a missing or outdated WSL installation when starting. Fixes [docker/for-win#13184](https://github.com/docker/for-win/issues/13184).
```

**Review by mat007 — COMMENTED** [review]
What about `_includes/desktop-install.html`? Should we comment out all but Windows in there?

**Comment by aevesdocker** [comment]
https://deploy-preview-16599--docsdocker.netlify.app/desktop/release-notes/ I removed the includes file and just added the Windows accordion. Looks the same in the preview

File: desktop/release-notes.md (hunk 2 above)
Comment (aevesdocker) [reply desktop/release-notes.md:60]: Perfect 👍

File: desktop/release-notes.md [hunk 3]
```diff
@@ -24,13 +24,77 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+            <i class="fa fa-chevron-down"></i>
+          </a>
+        </h5>
+      </div>
+      <div id="collapseOne" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne">
+        <div class="panel-body">
+          <a class="btn btn-primary" href="https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-win-amd64" role="button">Download file</a> 
+        <br>
+        <br>
+        <b>Checksum:</b>
```
Comment (mat007) [inline desktop/release-notes.md:48]:
```suggestion
        <b>Checksum: b9758c302b4bddd661a4b2307d9fa3f85658b5e69b3f11a0586f3dc32c524386</b> 
```

**Comment by mat007** [comment]
> I removed the includes file and just added the Windows accordion. Looks the same in the preview

Oh right! Will we keep these inline from now on?

**Comment by aevesdocker** [comment]
| Will we keep these inline from now on?

We could? But it makes the release-notes.md look extra complex when editing it. Having it in the includes file keeps everything neat, I think.

**Review by usha-mandya — APPROVED** [review]

File: desktop/release-notes.md [hunk 4]
```diff
@@ -24,13 +24,77 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## 4.16.3
+
+{% include release-date.html date="2023-01-26" %}
```
Comment (mat007) [inline desktop/release-notes.md:29]: Delta updates are still being created, we should probably postpone releasing until tomorrow.
```suggestion
{% include release-date.html date="2023-01-27" %}
```

File: desktop/release-notes.md [hunk 5]
```diff
@@ -24,13 +24,77 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+            <i class="fa fa-chevron-down"></i>
+          </a>
+        </h5>
+      </div>
+      <div id="collapseOne" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne">
+        <div class="panel-body">
+          <a class="btn btn-primary" href="https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-win-amd64" role="button">Download file</a> 
+        <br>
+        <br>
+        <b>Checksum: b9758c302b4bddd661a4b2307d9fa3f85658b5e69b3f11a0586f3dc32c524386</b>
```
Comment (mat007) [inline desktop/release-notes.md:48]: There is actually a new build now:
```suggestion
        <b>Checksum: 99f6600e7a05c7e56490600497b64a5039eb15f3413c1cf7b3a89bdfc4d947b8</b> 
```

**Comment by aevesdocker** [comment]
What time are you releasing @mat007

**Review by usha-mandya — APPROVED** [review]

**Pull request merged**